### DISCUSSION TITLE
[Bug] Setting gtk-titlebar to false results in the tab bar not being themed

### DISCUSSION THREAD

**Discussion by Gelmo** [ghostty-org/ghostty#3993] [Feature Requests, Ideas] [answered]
Config:
```
theme = Dracula
window-theme = ghostty
gtk-titlebar = false
```

With `gtk-titlebar` set to `true`, the tab bar and tabs respect the set theme. When set to `false`, the tab bar and tabs use the default theme. Note that `window-theme` is set to `ghostty` to ensure that the tabs use the set theme.

**Comment by 00-kat** [comment]
In case you (as in, anyone who has this issue and sees this comment) have both `window-decoration` and `gtk-titlebar` disabled, re-enabling `gtk-titlebar` will change nothing except for allowing the tabs to be themed.

(Not saying this isn't an issue.)

**Reply by Gelmo** [reply]
I do not have both disabled, as that results in, with i3, a missing highlight border for the window when focused. Having the former disabled and the latter enabled as you suggest, the Ghostty window has undesirable borders. I am only interested in using Ghostty with the config mentioned in the original post, as that is the only config that allows the window to be presented as desired.

**Reply by 00-kat** [reply]
That comment was more so directed to anyone else who finds this issue and happens to have both disabled; I knew you didn't because you didn't have it in the configuration you included in your post.

> as that results in, with i3, a missing highlight border for the window when focused

Interesting, I'll keep that in mind.

**Comment by Gelmo** [comment]
Updated the original post; I accidentally wrote `true` instead of `false` for the bug-producing configuration

**Comment by Gelmo** [comment]
Other discussions (for different bugs) that have mentioned this issue:
- https://github.com/ghostty-org/ghostty/discussions/3784
- https://github.com/ghostty-org/ghostty/discussions/4000

**Comment by Gelmo** [comment, marked as answer]
This has been fixed in 1.0.1
